walls.c

```diff
@@ -103,6 +104,27 @@ float get_front_sensors_error(void)
 	return distance[SENSOR_FRONT_LEFT_ID] - distance[SENSOR_FRONT_RIGHT_ID];
 }
 
+/**
+ * @brief Calculate and return the diagonal sensors error.
+ *
+ * Taking into account that robot is running between diagonal pillars, this
+ * function returns the distance is moved from the center of the diagonal.
```

Comment:
This description is not correct. It just tells whether or not we are too close to a pillar.

Reply:
4366a657

Reply:
> This function returns an error if the robot is too close to a pillar. The error is calculated as the difference between the sensed distance and a minimum distance threshold.

Reply:
acf79a2f